pkg/controller/volume/persistentvolume/controller.go

```diff
@@ -1212,6 +1212,11 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claimObj interfa
 		ClusterName:                   ctrl.clusterName,
 		PVName:                        pvName,
 		PVCName:                       claim.Name,
+
+		// will this need anything considering I see
+		//    Parameters:                    storageClass.Parameters,
+		// in https://github.com/kubernetes/kubernetes/pull/29006
+		// EBS plugin just accesses Parameters directly?
```

Comment:
I think this is unneeded because #29006 adds `Parameters:                    storageClass.Parameters,`